### ISSUE TITLE
Support Unix

### ISSUE THREAD

**Issue by aurexav** [hack-ink/AiR#40]

**Comment by johnblommers** [comment]
Note that we may install this project from source code following the instructions at the README. I already had Rust installed. After installing the two dependencies, it took only four minutes on my small form factor PC to successfully build AiR. It ran perfectly once I entered my OPENAI_API_KEY.

**Comment by AurevoirXavier** [comment]
Yep, most functions should work. But something relies on the OS API won't work, such as auto hide on lost focus. That needs to be fixed.
